src/CDDL-1.0.xml

```diff
@@ -0,0 +1,397 @@
+<spdx name="Common Development and Distribution License 1.0" identifier="CDDL-1.0" osi-approved="true">
+  <urls>
+    <url>http://www.opensource.org/licenses/cddl1</url>
+  </urls>
+  <notes>This license was released: 24 January 2004.</notes>
+  <license>
+    <title>
+      <p>COMMON DEVELOPMENT AND DISTRIBUTION LICENSE (CDDL) 
+        <br/>Version 1.0 
+      </p>
+    </title>
+    <body>
+      <list>
+        <li>
+          <b>1.</b>
+          <p>Definitions.</p>
+        </li>
+        <li>
+          <b>1.1.</b>
+          <p>&#x201C;Contributor&#x201D; means each individual or entity that creates or contributes to the
```

Comment:
The &#x201C; and &#x201D; correspond to HTML hex encoded left and right double quote characters respectively (see http://www.fileformat.info/info/unicode/char/201C/index.htm), which is consistent with the original source file. Whether we strictly need them versus just directly using the equivalent UTF-8 characters I do not know.